skiko/src/awtTest/kotlin/org/jetbrains/skiko/SkiaLayerPerformanceTest.kt

```diff
@@ -146,4 +199,30 @@ class SkiaLayerPerformanceTest {
             helpers.forEach { it.window.dispose() }
         }
     }
+
+    @Ignore("Doesn't pass with SOFTWARE API on macOS")
+    @Test
+    fun `FPS is near display refresh rate (multiple windows with clocks)`() = uiTest {
```

Comment:
This test throws AssertionError and it will be very flaky with the clocks content.

I suggest transform `printInfo` to `printInfo(assert = true)` and `printInfo(assert = false)`

Reply:
Good idea. I use this test locally to measure how changes affect rendering performance. What do you think, if we disable assertions, should we unignore it?

Reply:
yes, better to unignore